0001-kaa-protocol/README.md

```diff
@@ -7,83 +7,96 @@ contributors: Alexey Gamov <agamov@cybervisiontech.com>
 ---
 
 ## Introduction
-
-The document describes general guidelines, principles, and requirements for **Kaa protocol**.
+The document describes the general requirements, principles, design and guidelines for the **Kaa Protocol** (KP).
+KP is a standard protocol designed for connecting clients and endpoints (colloquially - devices) to a Kaa server.
 
 ## Requirements and constraints
-
 - Clients must be able to connect to Kaa server without using an SDK and with minimum custom implementation.
-- Encrypted and unencrypted channels should be supported.
-- Protocol should work over MQTT and CoAP.
-- Protocol must be extensible to support future extensions.
+- Encrypted and unencrypted channels must be supported.
+- KP should work over MQTT and CoAP.
```

Comment:
Can we define that MQTT must be supported and CoAP/etc should be supported?

This will allow fall-back to MQTT in case of e.g. CoAP version mismatch, etc.

--
Another point is MQTT version specification. At the moment Kaa requires 3.1.1 which is quite new and not widely available yet.